README.Rmd

```diff
@@ -58,7 +58,7 @@ lm_robust(y ~ z, data = sample_dat, clusters = my_cluster_var)
 difference_in_means(y ~ z, data = sample_dat, blocks = my_block_var)
 ```
 
-The [Getting Started](articles/estimatr-vignette.html) guide describes each estimator provided by **estimatr** and how it can be used in your analysis.
+The [Getting Started](http://estimatr.declaredesign.org/articles/estimatr-vignette.html) guide describes each estimator provided by **estimatr** and how it can be used in your analysis.
```

Comment:
I feel like the link text should be "Getting Started Guide" instead of just "Getting Started" although opinions may vary.

Reply:
Agree